docs/content/docs/deployment/resource-providers/yarn.md

```diff
@@ -184,6 +157,31 @@ The YARN session client also has a few "shortcut arguments" for commonly used se
 
 {{< top >}}
 
+### Per-Job Mode (deprecated)
```

Comment:
Replicate deprecation notice from the overview page

Reply:
Done.

Reply:
Done.